openwisp_radius/tests/test_api/test_freeradius_api.py

```diff
@@ -1132,14 +1132,65 @@ def test_accounting_filter_is_open(self):
 
     @freeze_time(START_DATE)
     def test_coova_accounting_on_200(self):
-        self.assertEqual(RadiusAccounting.objects.count(), 0)
+        nas_ip = "192.168.182.1"
```

Comment:
I recommend avoiding to change existing tests. This test verifies that a specific case works in a certain way, I'd leave it alone and add a new one if needed.